### ISSUE TITLE
Can I  take some terrain in Plotme plugin ?

### ISSUE THREAD

**Issue by GreatGBL** [WorldCretornica/PlotMe-Core#145]
Hello , I am small server owner , I use your plugin to create 256X256 side plotmeworld to give my players ;However , Most of players don't like it because It is flat , I decide to take some terrain on each plot, Can I do that ?

**Comment by MarkusAV** [comment]
In the future you will see possibilities for schematic support :)
But at the moment it is not possible

**Comment by MattBDev** [comment]
It is possible to build an add on that generates terrain but sadly, all the plugins i know that do this are privately made.

**Comment by ZachBora** [comment]
Ideas of things I want to add in a near future
* On claiming, load a schematic on the plot. Load either always specific schem or random schem
* Command to load a schematic on the plot.
* New generator called NoGenerator that will use vanilla generator and no roads.

**Comment by GreatGBL** [comment]
@ZachBora 
Actually，I spent 50$ to let others programmer help me add this function： when some new player take a new plot , in the center of plot will load a schem which I made ;However , If the schem is too large ,server will lag about 5-10s,everyone hate the lag,so I have no idea ,I only use it to make a old house,tree and well . Each new player can get those,

That is a picture here : http://attachment.mcbbs.net/forum/201407/06/234043doc5v04nbominld0.jpg.thumb.jpg

If you are Interest in it , you can join my server  or I send you this plugin ?  
I want to help you to add this future,Plotworld can load the schematic nature not only when player pick the new plot

**Comment by GreatGBL** [comment]
@MarkusAV 
@MattBDev 

:)

**Comment by ZachBora** [comment]
@GreatGBL We're planning a release tomorrow or saturday, but this weekend I can add this feature and it will be available for download from our jenkins.

**Comment by ZachBora** [comment]
@GreatGBL so just to understand correctly, they claim the plot and then a schematic (config for the path) gets pasted on top of the plot in the middle.

**Comment by GreatGBL** [comment]
@ZachBora  
Yep, When a player claim new plot , this plugin will pasted the schematic on the center;However , I wanna it is nature paste not when the player claim 

and sorry my English :(

**Comment by GreatGBL** [comment]
@ZachBora  you can join my server  to view
IP   112.80.16.202

**Comment by ZachBora** [comment]
I don't understand this part : 
"However , I wanna it is nature paste not when the player claim "

**Comment by GreatGBL** [comment]
@ZachBora 

Sorry my English :( 

my mean is if someone claim a plot,This plugin will paste the schematic (config for the path) on this plot .But In my ideal, I want to the plugin auto paste the schematic no matter someone clam in ,becasuse paste large schematic casuse server lag

**Comment by GreatGBL** [comment]
@ZachBora  
where are you living ? I will check your local time and watting for you  :)

**Comment by GreatGBL** [comment]
Sorry to my poor English ability and discourteous behaviour.
I really like your plugin :)  thx  !

**Comment by ZachBora** [comment]
@GreatGBL I have an idea, say it in your language and I'll try to translate it.

**Comment by GreatGBL** [comment]
Translation software is terrible , I try my best .

I have a small server （NO VIP）which has 50-200 players each day .The main server law ,everyone only build his house and farm in plotworld , normal world (minecraft default wolrd) is for mining or excavate. So I creae plotworld ,each plot has 256x256 ,it is so large that players has enough place to build his epic castle.However, most of players don't want to build in flat beacuse it lack terrain, but I don't want to use Resident or Worldguard to protect player's house the reason why both plugins has many issuse,for example some negetive players can destory the ourside area(no protect), and mining anywhere , building anywhere , like this 【http://xiangce.baidu.com/picture/detail/bf802bde29ab6700f0eb9d22902456f26a14f6fd?from=dialog#】

Therefor, I spend 4 days to make a beautiful terrain into schem  . In my decided ,when players claim the plot , the schematic gets pasted on top of the plot in the certer, I hope my player will be satisfaction .Although it is success paste ,the schem is too large (about 200x200) so server lag and slow when a player claim the plot , nobody like lag :(
So I change my mind , I make a very small schem like this 【http://attachment.mcbbs.net/forum/201407/06/234043doc5v04nbominld0.jpg.thumb.jpg】

In contrast, I don't like it ,I really hope for each plot has a terrain and enjoy minecraft 
My idea is when I build server and create plotworld , the schem automatic paste in each plot ,  not like my situation 【schem automatic paste when the player claim 】

All in all ,you can view my server or I send the has been changed plugin to you , for help you add this feature,
Thx for your reading !!

**Comment by ZachBora** [comment]
If I understand correctly, you would like that when the world generates, the plot already contain the schematic by default. This can be done.

**Comment by GreatGBL** [comment]
Aha yep , My means is it

**Comment by GreatGBL** [comment]
@ZachBora 
If you need the plugin(has been changed) , I can send to you for help . Plotme is the greatest plugin for ever !

**Comment by GreatGBL** [comment]
Thanks for your greate work . Can I use it in offline server (NO VIP) in 1.8.1 version ?

**Comment by MattBDev** [comment]
"(NO VIP)" is a term that I've never heard of so I have no idea what you mean by that. Offline server are not supported unless you use bungee or lillypad or something along those lines to authenticate users.

**Comment by GreatGBL** [comment]
"NO VIP“means is my server is free server ,player do not  necessary pay for anything .

My server used bungee,but bot bungee and server are offine , so I can not use the plugin ?

**Comment by MattBDev** [comment]
We don't support offline servers a.k.a "cracked" servers. It is an issue with protections and we are not allowed to support offline servers due to bukkit dev rules.

**Comment by GreatGBL** [comment]
How about this : bungess is online but server is offline

**Comment by MattBDev** [comment]
Yes. *facepalm*

**Comment by GreatGBL** [comment]
OK, Thanks for your great work !!

**Comment by GreatGBL** [comment]
Excuse me ,one more question , I want to know what the reason about unsupport with offine , hard match about UUID , your copyright  or Mojang's benfit ?

**Comment by ZachBora** [comment]
@GreatGBL please don't discuss offline in this issue, this is about Terrain not offline servers. There is already another issue open about offline here https://github.com/WorldCretornica/PlotMe-Core/issues/127
